src/plugins/cosmos/components/modals/Staking/views/UnstakeBroadcast.tsx

```diff
@@ -0,0 +1,157 @@
+import { InfoOutlineIcon } from '@chakra-ui/icons'
+import { Flex } from '@chakra-ui/layout'
+import { Button, Link, ModalCloseButton, Text as CText, Tooltip } from '@chakra-ui/react'
+import { CAIP19 } from '@shapeshiftoss/caip'
+import { Asset } from '@shapeshiftoss/types'
+import { chainAdapters } from '@shapeshiftoss/types'
+import { useForm } from 'react-hook-form'
+import { useTranslate } from 'react-polyglot'
+import { Amount } from 'components/Amount/Amount'
+import { MiddleEllipsis } from 'components/MiddleEllipsis/MiddleEllipsis'
+import { SlideTransition } from 'components/SlideTransition'
+import { Text } from 'components/Text'
+import { useModal } from 'context/ModalProvider/ModalProvider'
+import { BigNumber } from 'lib/bignumber/bignumber'
+
+type UnstakeBroadcastProps = {
+  assetId: CAIP19
+  cryptoUnstakeAmount: BigNumber
+  fiatRate: BigNumber
+  isLoading: boolean
+}
+
+export enum Field {
+  FeeType = 'feeType'
+}
+
+export type UnstakeBroadcastParams = {
+  [Field.FeeType]: chainAdapters.FeeDataKey
+}
+
+const DEFAULT_VALIDATOR_NAME = 'Shapeshift Validator'
```

Comment:
If we're in the unstake broadcast view, we will already have the validator info since we're unstaking one specific bonding, see: https://github.com/shapeshift/lib/blob/main/packages/chain-adapters/src/cosmossdk/CosmosSdkBaseAdapter.ts#L71

I'd expect the API for this component to be ready to receive it so we need minimal changes later on, but I'm fine with it like this for now, that'll take a minute or two to change when wiring up.

Reply:
done !